cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,16 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+    cy.get(':nth-child(1) > .form-control')
+      .type('luckezcy@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('testPassword1!');
+    cy.get('.btn')
```

Comment:
Here too, try to use a more specific selector. Selecting by '.btn' might lead to incorrect behavior if another button is added to the page in the future.